proposal/benchen216/README.rst

```diff
@@ -0,0 +1,64 @@
+==========================================
+Proposal for logic gate circuit simulation
+==========================================
+
+Basic information
+=================
+In recent days, FPGA SoC and some kinds of SoC can be use as AI or deep learning accessor,
+also 5g will use FPGA as RFSoc to extend the radio transmission.
+
+project URL:
+
+https://github.com/benchen216/logic-gate-sim
+
+
+Perspective users
+=================
+
+1. The student that feel verilog is so difficult.
+2. The people want to run simple logic circuit.
+
+System architecture
+===================
+
+Data Flow
+
+1. read in logic gate circuit syntax
+2. construct the circuit and fill to data type
+3. read in the input to run the circuit
+4. output the result
+
+API description
+===============
+
+1. logic gate circuit
+2. input value
+3. out put result
+4. performance trace
+
+Engineering infrastructure
+==========================
+
+1. circuit syntax parser
```

Comment:
The list looks more about your software system, rather than how you engineer the system.  You should add information about build tools, version controls, testing, CI, etc., in this section.